docs/topics/wf-swarm-dev-guide-additional-resources.adoc

```diff
@@ -0,0 +1,4 @@
+[[wf-swarm-additional-resources]]
+= Additional Resources
+
+* link:https://wildfly-swarm.gitbooks.io/wildfly-swarm-users-guide/content/v/2017.6.0/index.html[{WildFlySwarm} User Guide]
```

Comment:
We should probably also mention https://howto.wildfly-swarm.io/ and https://reference.wildfly-swarm.io/. My understanding is that the User Guide is being slowly phased out.